lib/src/androidTestDebug/java/com/nytimes/android/external/registerlib/GoogleProductResponseTest.java

```diff
@@ -0,0 +1,50 @@
+package com.nytimes.android.external.registerlib;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class GoogleProductResponseTest {
```

Comment:
Can we use assetJ for the assertions?
`assertThat(CODE.METHOD()).isNull()`
`assertThat(CODE.METHOD()).isNotNull()`